tests/fixtures/schema_with_defaults.json

```diff
@@ -0,0 +1,41 @@
+{
+    "type": "object",
+    "additionalProperties": false,
+    "properties": {
+        "age": {
+            "type": "number",
+            "default": 18
+        },
+        "name": {
+            "type": "string",
+            "default": "John Doe"
+        },
+        "nicknames": {
+            "type": "array",
+            "default": [
+                "yo",
+                "dawg"
+            ],
+            "items": {
+                "type": "string"
+            }
+        },
+        "pet": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "kind": {
+                    "type": "string",
+                    "default": "Dog"
+                }
+            },
+            "default": {
+                "kind": "Cat"
+            }
+        },
+        "profession": {
+            "type": "string",
+            "default": null
+        }
+    }
+}
```

Comment:
missing new line at the end of file

Reply:
ah, the only auto-generated file... sorry about that.

Reply:
fixed